.gitlab-ci.yml

```diff
@@ -105,13 +105,13 @@ variables:
     description: |-
       Version of mender-stress-test-client
   META_OPENEMBEDDED_REV:
-    value: "kirkstone"
+    value: "scarthgap"
     description: |-
       Version of meta-openembedded
   META_RASPBERRYPI_REV:
     value: "kirkstone"
     description: |-
-      Version of meta-raspberrypi
+      Version of meta-raspberrypi. Only applicable for kirkstone or older
```

Comment:
Is this enough, or will it break something?

Reply:
The build is turned off by default, so this variable will be ignored most of the cases and won't break anything. If the build is turned on, and `META_RASPBERRYPI_REV` would be `scarthgap`, then it would break.

In short, better to leave it as-is and rely on developer knowledge. We will get the rid of it in two years from now :angel: